fog/test-client/src/test_client.rs

```diff
@@ -710,10 +725,24 @@ impl TestClient {
                     }
                 }
             }
+            let transfer_duration = transfer_start.elapsed();
+            let sleep_duration = match period.checked_sub(transfer_duration) {
+                Some(duration) => duration,
+                None => {
+                    let excess_transaction_time = transfer_duration.sub(period);
+                    log::error!(
+                        self.logger,
```

Comment:
Should this be error? It will trigger an alert, right? Maybe warn instead?

Reply:
Done.